sdk/core/Azure.Core.TestFramework/src/Generated/Models/StartInformation.cs

```diff
@@ -25,5 +25,7 @@ public StartInformation(string xRecordingFile)
 
         /// <summary> Gets the x recording file. </summary>
         public string XRecordingFile { get; }
+        /// <summary> Gets or sets the x recording assets file. </summary>
+        public string XRecordingAssetsFile { get; set; }
```

Comment:
Should this model and serialize all the properties of the file rather than the path?

Reply:
I guess the proxy looks up the full data from the path? But agreed it might be simpler to just pass all the information.

Reply:
I mean, you could pass it but the test-proxy wouldn't understand it.

The reason we're passing an assets file is that you _won't_ know where the absolute path to the recordings is when they are externalized. You'll need to get them by relative + the root path described by the assets.json cloned directory.